### ISSUE TITLE
No access to noise_resids before fit ("prefit" noise resids)

### ISSUE THREAD

**Issue by pennucci** [nanograv/PINT#769]
@luojing1211:
I think noise_resids only exist after a call to fit_toas().  However, there should be a "prefit" noise realization, right?  This will be useful to have.

**Comment by paulray** [comment]
Probably @demorest should look at this...

**Comment by demorest** [comment]
Yeah the way the algorithm is structured you can't really get the noise resids without running a fit.  If you want "prefit" noise realizations you can run fit_toas() with all fit params disabled.
